R/derive_merged.R

```diff
@@ -370,46 +370,11 @@ derive_vars_merged_dt <- function(dataset,
                                   preserve = FALSE,
                                   check_type = "warning",
                                   duplicate_msg = NULL) {
-  assert_vars(by_vars)
-  dtc <- assert_symbol(enexpr(dtc))
-  filter_add <- assert_filter_cond(enexpr(filter_add), optional = TRUE)
-  assert_data_frame(dataset_add, required_vars = expr_c(by_vars, dtc))
-
-  deprecate_warn(
-    "0.8.0",
+  deprecate_stop(
+    "0.10.0",
```

Comment:
Should we change the version number?
The message is now
```
`derive_vars_merged_dt()` was deprecated in admiral 0.10.0 and is now defunct.
```
Shouldn't it be
```
`derive_vars_merged_dt()` was deprecated in admiral 0.8.0 and is now defunct.
```
?
@sadchla-codes , @bms63 , what do you think?

Reply:
@bundfussr it should have been 0.10.0? i'll make the updates

Reply:
`derive_vars_merged_dt()` was deprecated in admiral 0.8.0 and is now defunct. this look correct to me

Reply:
making the updates now!